examples/custom_swift_proto_compiler/do_test

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+# Use the Bazel binary specified by the integration test. Otherise, fall back
+# to bazel.
+bazel="${BIT_BAZEL_BINARY:-bazel}"
+
+# Generate Swift external deps and update build files
+"${bazel}" run //:tidy
+
+# Ensure that it builds
+"${bazel}" build //...
+
+# Ensure that Gazelle doesn't make any changes
+"${bazel}" run //:update_build_files -- --mode=diff
```

Comment:
We run `//:tidy` above. If any changes were to have happened, they would have before we get here. Do you want this above the `//:tidy` execution?

Reply:
I copied this from one of the other examples -- I can move it.